__tests__/homePage.test.jsx

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from '@testing-library/react'
```

Comment:
suggestion: test files can also be in TS ;)

Reply:
I was getting some kind of error from TS when I was setting this up so did .jsx. I switched it to .tsx just now and that error is gone.